electron-app/src/server/websites/aryion/aryion.service.ts

```diff
@@ -28,6 +28,7 @@ import { Website } from '../website.base';
 @Injectable()
 export class Aryion extends Website {
   readonly BASE_URL = 'https://aryion.com';
+  readonly MAX_CHARS: number = -1; // No limit
```

Comment:
nit: couldn't 'undefined' be considered the same?

Reply:
Yup, this makes sense to do - will update.